net/lcdringer/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2009-2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lcdringer
+PKG_VERSION:=0.0.2
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=W. Michael Petullo <mike@flyn.org>
+
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://www.flyn.org/projects/lcdringer
+PKG_MD5SUM:=ff3fe82593792c9111f957dd533369fb
+PKG_BUILD_DEPENDS:=+vala
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+
+define Package/lcdringer
+  SECTION:=network
+  CATEGORY:=Network
+  DEPENDS:=+libgee +libgstreamer1 +loudmouth
+  TITLE:=lcdringer
+  URL:=http://www.flyn.org/projects/lcdringer/
+  PKGARCH:=bcrm2708
```

Comment:
Does that ```PKGARCH:=bcrm2708``` work? I have only seen it as ```PKGARCH:=all``` a adeclaration that a compiled .ipk package can be installed to any architecture. But I don't remember ever seen that PKGARCH used to restrict building for other than the targeted platforms. 

I though that the the architecture restriction would be ```@BCRM2708 ``` on the depends line, or something like that.